public/index.js

```diff
@@ -2,15 +2,50 @@
   "use strict";
 
   addAnswersToSessionStorage();
+  updateAvatar('introduction', 'sleeping-lion');
+  updateAvatar('finish', 'finish__lion');
 
   $('.carousel').carousel({
     indicators: true,
     shift: 100
   });
 
   [
-   "send-email-button"
- ].forEach(function(button){
+    'lion',
+    'bear',
+    'monkey'
+  ].forEach(function (avatar) {
+    var node = document.getElementsByClassName(avatar)[0];
+    if (!node ) return;
+    node.addEventListener('click', function () {
+      var checkmark = document.getElementsByClassName('checkmark')[0];
```

Comment:
Are we looking for a checkmark inside the node? If so, do: `node.getElementsByClassName`. Since an unrelated thing with class checkmark will break this code